tensorboard/uploader/uploader_test.py

```diff
@@ -564,42 +576,6 @@ def test_upload_swallows_rpc_failure(self):
         uploader._upload_once()
         mock_client.WriteScalar.assert_called_once()
 
-    def test_upload_propagates_experiment_deletion(self):
-        logdir = self.get_temp_dir()
-        with tb_test_util.FileWriter(logdir) as writer:
-            writer.add_test_summary("foo")
-        mock_client = _create_mock_client()
-        uploader = _create_uploader(mock_client, logdir)
-        uploader.create_experiment()
-        error = test_util.grpc_error(grpc.StatusCode.NOT_FOUND, "nope")
-        mock_client.WriteScalar.side_effect = error
-        with self.assertRaises(uploader_lib.ExperimentNotFoundError):
-            uploader._upload_once()
-
-    def test_upload_preserves_wall_time(self):
```

Comment:
I decided to just plain delete this test. I think it's already covered by test_wall_time_precision().

Reply:
+1